canonical/lxd-ui
# move yaml switch to the footer of forms and flatten navigation

## Done

- move yaml switch to the footer of forms for instance, profile, network and storage pool create and edit
- flatten form navigations

Fixes WD-14519

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check yaml switch on instance, profile, network and storage pool creation and editing
